<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>barbarian</class>
 <widget class="QMainWindow" name="barbarian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
<<<<<<< HEAD
    <height>551</height>
=======
    <height>800</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
   </size>
  </property>
  <property name="windowTitle">
   <string>Barbarian</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>800</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
    <property name="currentIndex">
<<<<<<< HEAD
     <number>2</number>
=======
     <number>3</number>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
    </property>
    <widget class="QWidget" name="Search">
     <widget class="QPushButton" name="pushButton_Search_Packages">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Search">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Seach_Remotes">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Remotes to search in:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Remotes">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>271</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Help_Remotes">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>270</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
      <property name="iconSize">
       <size>
        <width>8</width>
        <height>8</height>
       </size>
      </property>
     </widget>
<<<<<<< HEAD
     <widget class="QLabel" name="label_Spinner">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_Progress">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>360</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
=======
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
    </widget>
    <widget class="QWidget" name="Installed">
     <widget class="QListWidget" name="listWidget_Installed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>781</width>
        <height>441</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_Installed">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>5</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Installed</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Remove">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>781</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Preferences">
     <widget class="QLabel" name="label_Preferences">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Preferences</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_Preferences">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>781</width>
        <height>491</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_General">
       <attribute name="title">
        <string>General</string>
       </attribute>
<<<<<<< HEAD
       <widget class="QLabel" name="label_Conan_Dir">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Conan Directory:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Conan_Dir">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Conan_Dir_Help">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>30</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Save_BuildSystem">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>250</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_BuildSystem">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>141</width>
=======
       <widget class="QLabel" name="label_Remotes">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>60</y>
          <width>71</width>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <height>20</height>
         </rect>
        </property>
        <property name="text">
<<<<<<< HEAD
         <string>Default Build System:</string>
=======
         <string>Remotes:</string>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QLineEdit" name="lineEdit_BuildSystem">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>250</y>
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Profile">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>100</y>
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Profile">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>100</y>
          <width>61</width>
=======
       <widget class="QTextEdit" name="textEdit_Remotes">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>491</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_Config">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>260</y>
          <width>491</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Config">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>71</width>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <height>20</height>
         </rect>
        </property>
        <property name="text">
<<<<<<< HEAD
         <string>Profile:</string>
=======
         <string>Config:</string>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QPushButton" name="pushButton_Profile_Save">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>100</y>
          <width>41</width>
          <height>23</height>
=======
       <widget class="QPushButton" name="pushButton_Remotes">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>220</y>
          <width>51</width>
          <height>21</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QPushButton" name="pushButton_Storage_Save">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>180</y>
          <width>41</width>
          <height>23</height>
=======
       <widget class="QPushButton" name="pushButton_Config">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>440</y>
          <width>51</width>
          <height>21</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QLineEdit" name="lineEdit_Storage_Path">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_Storage_Path">
        <property name="geometry">
         <rect>
          <x>46</x>
          <y>180</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Storage Path:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_Comp_Lev">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>121</width>
=======
       <widget class="QLabel" name="label_Conan_Dir">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <height>20</height>
         </rect>
        </property>
        <property name="text">
<<<<<<< HEAD
         <string>Compression Level:</string>
=======
         <string>Conan Directory:</string>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QLineEdit" name="lineEdit_Comp_Lev">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>330</y>
=======
       <widget class="QLineEdit" name="lineEdit_Conan_Dir">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QPushButton" name="pushButton_Comp_Save">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>330</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Storage_Path_Help">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>180</y>
=======
       <widget class="QPushButton" name="pushButton_Conan_Dir_Help">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>30</y>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </widget>
<<<<<<< HEAD
      <widget class="QWidget" name="tab_advanced">
       <attribute name="title">
        <string>Advanced</string>
       </attribute>
       <widget class="QLabel" name="label_Config">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>180</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Config:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_Remotes">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>71</width>
=======
      <widget class="QWidget" name="tab_build">
       <attribute name="title">
        <string>Build</string>
       </attribute>
       <widget class="QLabel" name="label_BuildSystem">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>141</width>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
          <height>20</height>
         </rect>
        </property>
        <property name="text">
<<<<<<< HEAD
         <string>Remotes:</string>
=======
         <string>Default Build System:</string>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QTextEdit" name="textEdit_Config">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>491</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Config">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>340</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_Remotes">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>491</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Remotes">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>150</y>
          <width>51</width>
          <height>21</height>
=======
       <widget class="QLineEdit" name="lineEdit_BuildSystem">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>491</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Save_BuildSystem">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>30</y>
          <width>51</width>
          <height>23</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Package_List">
     <widget class="QPushButton" name="pushButton_Package_Install">
      <property name="geometry">
       <rect>
<<<<<<< HEAD
        <x>580</x>
        <y>340</y>
        <width>171</width>
        <height>20</height>
=======
        <x>619</x>
        <y>480</y>
        <width>171</width>
        <height>23</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
       </rect>
      </property>
      <property name="text">
       <string>Install</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_PackageList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
<<<<<<< HEAD
        <width>741</width>
        <height>291</height>
=======
        <width>781</width>
        <height>431</height>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Package_List_Back">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Search_Result">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Search Result</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_Project_Directory">
      <property name="geometry">
       <rect>
        <x>10</x>
<<<<<<< HEAD
        <y>340</y>
=======
        <y>480</y>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Project Directory:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Project_Directory">
      <property name="geometry">
       <rect>
        <x>130</x>
<<<<<<< HEAD
        <y>340</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Installation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>380</y>
        <width>741</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Project_Directory_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>110</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>(Full Path)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_Installation">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>40</y>
        <width>31</width>
        <height>461</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
=======
        <y>480</y>
        <width>481</width>
        <height>23</height>
       </rect>
      </property>
>>>>>>> f792dea0103330dac4bf35032dc2fb13f4df1e3e
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionInstalled"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_QT"/>
    <addaction name="actionAbout_Conan"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionInstalled">
   <property name="text">
    <string>Installed</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionAbout_Conan">
   <property name="text">
    <string>About Conan</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
